src/components/footer/footer.jsx

```diff
@@ -0,0 +1,56 @@
+// Bootstrap CSS
+import "bootstrap/dist/css/bootstrap.min.css";
+// Bootstrap Bundle JS
+import "bootstrap/dist/js/bootstrap.bundle.min";
+
+import logo from './2.png';
```

Comment:
toutes les images doivent être dans `assets/images` et le nom de l’image doit être plus lisible.
```suggestion
import logo from 'assets/images/logo.png';
```